include/z64actor.h

```diff
@@ -515,6 +515,46 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+// A set of 4 triangles which appear as a ring around an actor when the player Z-Targets it
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ f32 radius; // distance towards the center of the locked on actor
+    /* 0x10 */ Color_RGB8 color;
+} LockOnReticle; // size = 0x14
+
+typedef struct {
+    /* 0x00 */ Vec3f naviPos; // Used by Navi to indicate a targetable actor or general hint
+    /* 0x0C */ Vec3f lockOnPos;
+    /* 0x18 */ Color_RGBAf naviInnerColor;
+    /* 0x28 */ Color_RGBAf naviOuterColor;
+    /* 0x38 */ Actor* naviHoverActor; // The actor that Navi hovers over
+    /* 0x3C */ Actor* lockOnActor;
+    /* 0x40 */ f32 naviMoveProgressFactor; // Controls Navi so she can smootly transition to the target actor
+    /* 0x44 */ f32 reticleRadius; // Control the reticle coming in from offscreen when you first target
+    /* 0x48 */ s16 reticleFadeAlphaControl; // Set and fade the reticle alpha. Also controls the reticle drawing with non-zero values.
+    /* 0x4A */ u8 naviActorCategory;
+    /* 0x4B */ u8 rotZTick;
+    /* 0x4C */ s8 lockOnIndex;
+    /* 0x50 */ LockOnReticle lockOnReticles[3];
+    /* 0x8C */ Actor* forcedTargetActor; // Never set to non-NULL
+    /* 0x90 */ Actor* bgmEnemy; // The nearest enemy to player with the right flags that will trigger NA_BGM_ENEMY
+    /* 0x94 */ Actor* arrowPointedActor;
+} TargetContext; // size = 0x98
```

Comment:
```suggestion
    /* 0x28 */ Color_RGBAf naviOuterColor;
    /* 0x38 */ Actor* naviHoverActor; // The actor that Navi hovers over
    /* 0x3C */ Actor* lockOnActor;
    /* 0x40 */ f32 naviMoveProgressFactor; // Controls Navi so she can smootly transition to the target actor
    /* 0x44 */ f32 reticleRadius; // Control the reticle coming in from offscreen when you first target
    /* 0x48 */ s16 reticleFadeAlphaControl; // Set and fade the reticle alpha. Also controls the reticle drawing with non-zero values.
    /* 0x4A */ u8 naviHoverActorCategory; // category of the actor Navi is currently hovering over
    /* 0x4B */ u8 reticleSpinCounter; // counts up when a reticle is active, used for z rotation of reticle
    /* 0x4C */ s8 lockOnIndex;
    /* 0x50 */ LockOnReticle lockOnReticles[3];
    /* 0x8C */ Actor* forcedTargetActor; // Not used, always set to NULL
    /* 0x90 */ Actor* bgmEnemy; // The nearest enemy to player with the right flags that will trigger NA_BGM_ENEMY
    /* 0x94 */ Actor* arrowHoverActor; // actor a target arrow is currently hovering over
} TargetContext; // size = 0x98
```
Open for discussions ofc, but this is the direction id like to see some of these names move in

Reply:
All of these seem good to me